docs/source/dev_environment_setup.md

```diff
@@ -65,6 +65,8 @@ will try to find. Currently only `opendiff` is listed there (but we may add addi
 manually add it to this file. See the ApprovalTests [README](https://github.com/approvals/ApprovalTests.Python) for
 examples (illustrative particularly for file paths on Windows).
 
+!!! warning
+    The project root contains an env file (`development_environment`), in which are default values for _all_ of the environment variables used to control and configure the FlowDB, FlowAPI, FlowMachine and FlowAuth. You will need to source this file, or otherwise set the environment variables before running _any other command_.
```

Comment:
```suggestion
    The project root contains an env file (`development_environment`), in which are default values for _all_ of the environment variables used to control and configure the FlowDB, FlowAPI, FlowMachine and FlowAuth. You will need to source this file (`set -a && . ./development_environment && set +a`), or otherwise set the environment variables before running _any other command_.
```